gstools/variogram/estimator.pyx

```diff
@@ -106,21 +105,21 @@ cdef inline double estimator_cressie(const double f_diff) nogil:
 ctypedef double (*_estimator_func)(const double) nogil
 
 cdef inline void normalization_matheron(
-    vector[double]& variogram,
-    vector[long]& counts
+    double[:] variogram,
+    long[:] counts,
```

Comment:
I thought `vector` is faster? And is this still using the references? (formerly marked with `&`) Or is this done automatically by inlining the function? Again: I could use some training in cython.

Reply:
It is faster, but I think only during the initialization of the variable. It does not call any Python code, in contrast to the `cdef double[:, :] variogram = np.zeros(...)`. But I'm very sure that that is negligible and I think the code is a lot cleaner when only using Cython's memoryviews and not mixed up with C++ data types.

Reply:
Nice.